runtime/src/sdk.rs

```diff
@@ -385,3 +385,8 @@ pub unsafe extern "C" fn extism_log_file(
     }
     true
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn extism_version() -> *const u8 {
+    env!("CARGO_PKG_VERSION").as_ptr()
```

Comment:
Is this the right manifest to grab the version from? it seems we are on release `v0.0.1-rc.4` but this one is showing `rc.3`.

Reply:
My mistake. We identified my main branch was pointed to the wrong origin.